lib/RateLimiterStoreAbstract.js

```diff
@@ -268,7 +233,7 @@ module.exports = class RateLimiterStoreAbstract extends RateLimiterAbstract {
           }
         })
         .catch((err) => {
-          this._handleError(err, 'get', resolve, reject, key, options);
+          this._handleError(err, 'get', resolve, reject, [key, options]);
```

Comment:
@florian-schunk Should it reject like other methods?

Reply:
So it calls `_handleError` from RateLimiterInsuredAbstract. It makes sense. Thank you.